_docs/setup/kubernetes/quick-start.md

```diff
@@ -32,7 +32,15 @@ support).
 
 * Depending on your Kubernetes provider:
 
-### [Minikube](https://github.com/kubernetes/minikube/releases)
+  * To install Istio locally, install the latest version of
+[Minikube](https://kubernetes.io/docs/getting-started-guides/minikube/) (version 0.25.0 or later).
+  ```
```

Comment:
Please add "bash" after the triple ticks.

Reply:
I will update this, sec